sihl-web/test/csrf.ml

```diff
@@ -1,6 +1,8 @@
 open Alcotest_lwt
 open Lwt.Syntax
 
+let token_alco = Alcotest.testable Sihl_type.Token.pp Sihl_type.Token.equal
```

Comment:
Do you generally want to remove these from the model?

Reply:
Yes, `pp` and `equal` are enough to create the alcotest instance quickly yourself.